network-api/app/networkapi/management/commands/heroku_release.py

```diff
@@ -14,6 +14,4 @@ def handle(self, *args, **options):
 
         if settings.LOAD_FIXTURE:
             call_command('loaddata', 'app/networkapi/fixtures/test_data.json')
-            site = Site.objects.get(id=1)
-            site.domain = 'localhost:8000'
-            site.save()
+            call_command('update_site_domain')

```

Comment:
Why do we need this for Heroku? Shouldn't it be a different url?

Reply:
The fixture data relates pages to sites via domain, strangely, and is currently set to use `localhost:8000`

This will make sure the pages loaded from the fixture are visible in the admin interface, and shouldn't affect anything beyond that. If it does, we can try using the HEROKU_APP_NAME variable to update the site and pages in the release command.